apis/stacks/v1alpha1/stackdefinition.go

```diff
@@ -27,11 +27,9 @@ import (
 
 // StackDefinitionSpec defines the desired state of StackDefinition
 type StackDefinitionSpec struct {
-	AppMetadataSpec `json:",inline"`
-	CRDs            CRDList         `json:"customresourcedefinitions,omitempty"`
-	Controller      ControllerSpec  `json:"controller,omitempty"`
-	Permissions     PermissionsSpec `json:"permissions,omitempty"`
-	Behavior        Behavior        `json:"behavior,omitempty"`
+	StackSpec `json:",inline"`
```

Comment:
Has this already been tested? If I recall correctly, when I tried this in the past there was an issue with nesting multiple levels of `",inline"`.

Reply:
StackDefinition and the resulting Stack stack-wordpress both reconciled for me with this.